store/doi.js

```diff
@@ -19,7 +19,8 @@ class DOI extends Store {
         : null
 
     if (lag != null && lag < 0) {
-      console.error('DOI enrichment count is less then 0!', lag)
+      if (process.env.NODE_ENV === 'development')
+        console.error('DOI enrichment count is less then 0!', lag)
```

Comment:
Maybe it would be worth to trowing it and log it into [Sentry](https://github.com/oacore/dashboard/blob/master/pages/_app/index.jsx#L92)?

Reply:
How Sentry would work on development? Will it log to console?

I would like to log to console during development and report to Sentry if production.

Reply:
AFAIK it logs it always. Basically you will see something like `Uncaught in promise Error: <error-message> and stacktrace`. I didn't test it. So maybe it doesn't get caught at all by the even listener.